packages/odyssey-react-mui/src/NativeSelect.tsx

```diff
@@ -126,6 +140,21 @@ const NativeSelect: ForwardRefWithType = forwardRef(
         uncontrolledValue: defaultValue,
       })
     );
+    const inputRef = useRef<HTMLSelectElement>(null);
+
+    useImperativeHandle(
+      inputFocusRef,
+      () => {
+        const element = inputRef.current;
```

Comment:
I don't think we need to create this `element` variable.

I'm fine having `inputRef.current?.focus()` below.